src/class-parsely.php

```diff
@@ -406,44 +416,44 @@ public function convert_jsonld_to_parsely_type( string $type ): string {
 	}
 
 	/**
-	 * Determines if an API key is saved in the options.
+	 * Determines if an Site ID is saved in the options.
 	 *
 	 * @since 2.6.0
 	 *
-	 * @return bool True is API key is set, false if it is missing.
+	 * @return bool True is Site ID is set, false if it is missing.
```

Comment:
For all the functions that have been renamed, it would be nice to have a `@since` tag, stating that the function has been renamed. Let's leave this for last, as we'll need to determine if we will release this in 3.7 or 3.8 (to be discussed).